<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MemEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="baseAddress">
         <property name="placeholderText">
          <string>Base</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currAddress">
         <property name="placeholderText">
          <string>Cursor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value">
         <property name="placeholderText">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="scanType">
         <item>
          <property name="text">
           <string>int8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>int16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>int32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>float16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>float32</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
